Cargo.lock

```diff
@@ -3428,6 +3428,7 @@ dependencies = [
  "rustc_ast",
  "rustc_span",
  "rustc_target",
+ "smallvec 1.4.2",
```

Comment:
specifying a version number here works :O

Reply:
I didn't do this by hand :) I think :D